src/Umbraco.Web.UI.Client/src/views/common/dialogs/help.html

```diff
@@ -1,13 +1,13 @@
 <div class="umb-panel" ng-controller="Umbraco.Dialogs.HelpController">
     <div class="umb-panel-header">
-        <h1 class="headline">Help</h1>
+        <h1 class="headline"><localize key="sections_help">Help</localize></h1>
         <small class="umb-version">Umbraco version {{version}}</small>
     </div>
 	
 	<div class="umb-panel-body umb-scrollable">
 		<div class="tab-content umb-control-group">
 			<div class="umb-pane">
-				<h5>Help topics for: {{section}}</h5>
+				<h5><localize key="help_helpTopicsFor">Help topics for</localize>: {{section}}</h5>
```

Comment:
Okay, I think I saw it used it like that somewhere but not sure :)
If not it should be fairly easy to use localizationService here :) https://github.com/umbraco/Umbraco-CMS/blob/7c4a189aa3cf583954defd9c43a3e55e325f2c3f/src/Umbraco.Web.UI.Client/src/views/common/dialogs/help.controller.js